test/src/queryhandler.spec.ts

```diff
@@ -497,5 +488,11 @@ describe('Query handler tests', function () {
       });
       expect(value).toMatchObject({textContent: 'world', tagName: 'BUTTON'});
     });
+
+    it('should work with commas', async () => {
+      const {page} = getTestState();
+      const elements = await page.$$('div, ::-p-text(world)');
```

Comment:
what takes precedence over comma? e.g., how is the following case treated `div, p >>> ::-p-text(world)`

Reply:
div  >>> ::-p-text(world), p >>> ::-p-text(world)

Reply:
:is(div, p)

Reply:
Let's add tests:

1) div, p >>> ::-p-text(world)
2) div, p the right order